tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_redhat_inventory.yaml

```diff
@@ -31,21 +31,3 @@
             value: NVD_JSON_PATH
         - update_interval:
             value: '5s'
```

Comment:
Maybe we should keep these modules turned off, to improve the stability of the test

Reply:
Fixed